### PR TITLE
Added manage controls for list of speakers of agenda items.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Fixed typo (speach -> speech)

### CODE REVIEW

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 1]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
 
 
 <h3 translate>List of speakers
-  <span ng-if="item.speakerListClosed" class="label label-danger" translate>closed</span>
   <span os-perms="agenda.can_manage">
     <button ng-if="item.speakerListClosed" ng-click="closeList(false)"
-        class="btn btn-sm btn-default" translate>
-      Open list
+        class="btn btn-sm btn-danger" translate>
+      <b>Closed</b>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please remove the b-tag

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 2]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
     </button>
     <button ng-if="!item.speakerListClosed" ng-click="closeList(true)"
-        class="btn btn-sm btn-default" translate>
-      Close list
+        class="btn btn-sm btn-success" translate>
+      <b>Opened</b>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please remove the b-tag

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 3]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
-   * start/stop speech
-   * button 'put/remove me on/from the list'
-   * check permissions
+   * show only 'add me' OR 'remove me' button
 -->
-<ol>
-  <li ng-repeat="speaker in item.speakers">
+<div class="well">
+  <p>
+  <b translate>Old speakers:</b>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please remove the b-tag

Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: The omitted closing p-tag is ended after  the button. So you should add white-spaces before the button.

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 4]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
+        {{ speaker.user.get_full_name() }}
+        <small class="grey">
+          [{{speaker.begin_time | date:'yyyy-MM-dd HH:mm:ss'}} –
+          {{speaker.end_time | date:'yyyy-MM-dd HH:mm:ss'}}]
+        </small>
+    </ol>
+  </div>
+
+  <p>
+  <b translate>Current speaker:</b>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please remove the b-tag

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 5]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
+  <b ng-repeat="speaker in item.speakers | filter: {end_time: null, begin_time: '!!'}">
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: please remove the b-tag

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 6]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
-    <button os-perms="agenda.can_manage" ng-click="removeSpeaker(speaker.id)"
-        class="btn btn-default btn-xs">
-      <i class="fa fa-times"></i>
-    </button>
-  </li>
-</ol>
+  </b>
+
+  <p>
+  <b translate>Next speakers:</b>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: please remove the b-tag

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 7]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
+  <ol>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please add white-spaces to make clear where the p-tag ends.

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 8]
```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
...
+      {{ speaker.user.get_full_name() }}
+      <button os-perms="agenda.can_manage" ng-click="removeSpeaker(speaker.id)"
+          class="btn btn-default btn-xs">
+        <i class="fa fa-times"></i>
+      </button>
+      <button os-perms="agenda.can_manage" ng-click="beginSpeech(speaker.id)"
+          class="btn btn-default btn-xs">
+        <i class="fa fa-play"></i>
+      </button>
+    </li>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please remove the closing li-tag

**Comment by ostcar** [comment]
Some comments in the html-code. Then it can be merged

**Comment by emanuelschuetze** [comment]
Done

**Comment by ostcar** [comment]
Please rebase and merge

**Pull request merged**